policy/modules/system/systemd.te

```diff
@@ -402,6 +402,8 @@ storage_raw_read_fixed_disk(systemd_generator_t)
 
 systemd_log_parse_environment(systemd_generator_t)
 
+term_dontaudit_use_unallocated_ttys(systemd_generator_t)
```

Comment:
Youre not allowing it here as you suggest in the topic. Youre silently block it instead. I would probably *allow* this access.

Reply:
I forgot to remove this one from the PR, sorry about that. And indeed we shoudl allow this instead :)